app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -267,13 +267,27 @@ function buildArticleHTML(article) {
         '</div></a>';
     }
 
-    return (
-      '<article class="crayons-story" data-article-path="' +
+    var navigationLink = `
```

Comment:
refactor (non-blocking): You can use [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).